frontend/src/app/supportAdmin/SupportAdmin.tsx

```diff
@@ -8,7 +8,7 @@ const SupportAdmin = () => {
           <div className="prime-container card-container">
             <div className="usa-card__header">
               <div>
-                <h2>Support admin</h2>
```

Comment:
Changing to an `<h1>` changed the style... not sure if this matters since this is used by support admin/devs but wanted to flag this

Before:
<img width="1185" alt="Screen Shot 2022-11-21 at 10 57 34" src="https://user-images.githubusercontent.com/20211771/203115256-b1db75b6-2c4b-4197-9692-566cc6306e55.png">

After:
<img width="1166" alt="Screen Shot 2022-11-21 at 10 57 38" src="https://user-images.githubusercontent.com/20211771/203115514-62be7db6-54f2-433c-be09-0309481cdadb.png">

Reply:
looks good to me now! thank you for fixing this!